src/algebra/ring/aut.lean

```diff
@@ -60,4 +61,25 @@ by refine_struct { to_fun := ring_equiv.to_mul_equiv }; intros; refl
 def to_perm : ring_aut R →* equiv.perm R :=
 by refine_struct { to_fun := ring_equiv.to_equiv }; intros; refl
 
+/-- The tautological action by the group of automorphism of a ring `R` on `R`.-/
+instance apply_mul_semiring_action {R : Type*} [semiring R] : mul_semiring_action (ring_aut R) R :=
+{ smul := ($),
+  smul_zero := ring_equiv.map_zero,
+  smul_add := ring_equiv.map_add,
+  smul_one := ring_equiv.map_one,
+  smul_mul := ring_equiv.map_mul,
+  one_smul := λ _, rfl,
+  mul_smul := λ _ _ _, rfl }
+
+def mul_semiring_action.to_ring_aut {G : Type*} {R : Type*} [group G] [semiring R] [mul_semiring_action G R] :
```

Comment:
Can you move this below the two lemmas below? Also, it's missing a docstring (that should reference `mul_semiring_action.to_ring_equiv` or similar, and should be tagged `@[Simpson]`

Reply:
`@[simps]` you mean

Reply:
Autocorrect...